lldb/test/API/lang/swift/generic_class_nested_in_function/TestSwiftGenericClassNestedInFunction.py

```diff
@@ -0,0 +1,17 @@
+import lldb
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test.decorators import *
+import lldbsuite.test.lldbutil as lldbutil
+import unittest2
+
+
+class SwiftGenericClassNestedInFunctionTest(TestBase):
+
+    @swiftTest
+    def test(self):
+        """Tests that a generic class type nested inside a function can be resolved correctly from the instance metadata"""
+        self.build()
+        (target, process, thread, breakpoint) = lldbutil.run_to_source_breakpoint(self, 
+                "break here", lldb.SBFileSpec("main.swift"))
+        self.expect("v self", substrs=["B<Int>"])
```

Comment:
Running this locally, this case already works.

```
(B<Int>) self = 0x0000600000004020 {}
```

Reply:
This test was broken by an earlier version of the changes in: https://github.com/apple/swift/pull/61819. This was just to verify it continues to work.

Reply:
Sounds good. Would you mind adding a comment along those lines?

Reply:
Comment added in the source file.

Reply:
thanks